crates/aptos-rosetta/src/block.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    common::{check_network, get_timestamp, handle_request, with_context},
+    error::{ApiError, ApiResult},
+    types::{
+        Block, BlockIdentifier, BlockRequest, BlockResponse, Transaction, TransactionIdentifier,
+    },
+    RosettaContext,
+};
+use aptos_crypto::HashValue;
+use aptos_logger::debug;
+use std::str::FromStr;
+use warp::Filter;
+
+pub fn routes(
+    server_context: RosettaContext,
+) -> impl Filter<Extract = impl warp::Reply, Error = warp::Rejection> + Clone {
+    warp::post().and(
+        warp::path!("block")
+            .and(warp::body::json())
+            .and(with_context(server_context))
+            .and_then(handle_request(block)),
+    )
+}
+
+async fn block(request: BlockRequest, server_context: RosettaContext) -> ApiResult<BlockResponse> {
+    debug!("/block");
```

Comment:
trace log?

Reply:
Yeah, missed the trace logs, will add them.